packit_service/models.py

```diff
@@ -650,6 +650,12 @@ def get_branch_name(self) -> Optional[str]:
             return trigger_object.name
         return None
 
+    def get_release_tag(self) -> Optional[int]:
+        trigger_object = self.job_trigger.get_trigger_object()
+        if isinstance(trigger_object, ProjectReleaseModel):
+            return trigger_object.tag_name
```

Comment:
tag_name is an int? interesting and confusing to me

Reply:
Ups. I've copy-pasted an already existing method..;) Will fix that..;)

Reply:
And mypy does not complain?

Reply:
Fixed.